src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+$main: #060b35;
+$secondary: #616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  flex-direction: column;
+  width: 200px;
+  border: solid 1px #f3f3f3;
+  display: flex;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin: 0 0 4px;
+  color: $main;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $secondary;
+}
+
+.card__reviews {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  line-height: 140%;
+}
+
+.card__reviews-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  margin-bottom: 16px;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: $secondary;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: $main;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  font-style: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  border: 1px solid #00acdc;
+  color: white;
+}
+
+.card__button:hover {
+  background-color: #ffff;
+  color: #00acdc;
+}
+
+.stars--1 > .stars__star:first-child,
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(n) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
you can also try to make it work with SCSS @each - it will loop through all stars, if setted up correctly 😉